### PR TITLE
Remove Jitpack support

### PR DESCRIPTION
**Pull request by rawbee**
## Description of the change

Deprecate Jitpack support. While removing it from the docs doesn't actually disable Jitpack from working, we don't want to encourage customers to use it any more.

### CODE REVIEW

**Comment by TrekSoft** [comment]
Looks like the workflow checks need to be updated from your last change moving the API secret?

File: settings.gradle [hunk 1]
```diff
@@ -1,2 +1,3 @@
 include ':pinwheel-android'
+include ':app'
```
Comment (TrekSoft) [inline settings.gradle:2]: Did you intend to leave this here or was this leftover from local testing?

Comment (rawbee) [reply settings.gradle:2]: Adding this back is partly the reason for deprecating Jitpack support.

**Comment by rawbee** [comment]
@TrekSoft working on updating CircleCI to work with the new api secret setup to fix the tests.

**Review by TrekSoft — APPROVED** [review]
cat-14-3

**Pull request merged**